### PR TITLE
add candy v3 errors

### PR DESCRIPTION
**Pull request by MarkSackerberg**
As discussed via discord adding the two new programs to `metaboss find error`

### CODE REVIEW

**Comment by MarkSackerberg** [comment]
Currently there is an issue because CM v2 and core error enums have the same names. I'd like to find a solution before merging.

If you have a proposal please let me know.

File: src/decode.rs [hunk 1]
```diff
@@ -354,6 +354,8 @@ fn parse_key(key: Key) -> String {
         Key::EditionMarker => String::from("EditionMarker"),
         Key::UseAuthorityRecord => String::from("UseAuthorityRecord"),
         Key::CollectionAuthorityRecord => String::from("CollectionAuthorityRecord"),
+        Key::TokenOwnedEscrow => String::from("TokenOwnedEscrow"),
```
Comment (samuelvanderwaal) [inline src/decode.rs:357]: You'll need to bump `mpl-token-metadata` version to 1.6 to support this.

**Comment by samuelvanderwaal** [comment]
> Currently there is an issue because CM v2 and core error enums have the same names. I'd like to find a solution before merging.
> 
> If you have a proposal please let me know.

I don't quite follow. It looks like it works?

**Comment by MarkSackerberg** [comment]
> I don't quite follow. It looks like it works?

yes - because that comment of mine is outdated already. Added an ugly If - else statement to decode.rs to make it work.

**Comment by samuelvanderwaal** [comment]
Once you bump to mpl-token-metadata v1.6 the rest of the errors should go away and it should work:

Screen Shot 2022-10-26 at 11 58 19 PM

**Pull request merged**